docs/userguide.rst

```diff
@@ -24,14 +24,27 @@ Basic usage
 .. code-block:: console
 
     $ # download twiki
-    $ twikiget https://twiki.cern.ch/twiki/bin/view/Main/ZhuTopAnalysis
-    $ # list downloaded files
+    $ twikiget archive https://twiki.cern.ch/twiki/bin/view/Main/ZhuTopAnalysis
     $ ls
     ZhuTopAnalysis.warc cache
+    $ # once the twiki is archived we can list it's contents:
+    $ twikiget list ZhuTopAnalysis.warc
+    $ ...
+    $ # we can also view the raw content of each file:
+    $ twikiget view ZhuTopAnalysis.warc https://twiki.cern.ch/twiki/bin/view/Main/ZhuTopAnalysis
+    $ ...
```

Comment:
Works well :+1: I left some cosmetic comments to the help messages. Please also check formatting of multi-line docs, there are some troubles.